server/models/Expense.js

```diff
@@ -480,15 +481,26 @@ function defineModel() {
     return legacyPayoutMethod === 'paypal' ? PayoutMethodTypes.PAYPAL : PayoutMethodTypes.OTHER;
   };
 
-  Expense.getMostPopularExpenseTagsForCollective = async function (collectiveId, limit = 100) {
+  Expense.getCollectiveExpensesTags = async function (collectiveId, limit = 100) {
```

Comment:
I would keep the original `getMostPopularExpenseTagsForCollective` since it is relatively simpler to run.

Reply:
Done